user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -1900,7 +1902,11 @@ define([
   function customSort(response) {
 
     var sortFunction = function (a, b) {
-     return window.parent.localeCompare(a.file.name, b.file.name);
+      //if the file doesn't have a title, sort by its name
+      //this is expected to only be the case for root folders
+      let aCompare = a.file.title ? a.file.title : a.file.name;
```

Comment:
Better as `const aCompare = a.file.title || a.file.name;`

Reply:
ah... delicious js syntactic sugar... wasn't aware of this variety 😋